### ISSUE TITLE
Release/version-update automation (maybe `cargo-release`)

### ISSUE THREAD

**Issue by sourcefrog** [sourcefrog/cargo-mutants#213]

**Comment by kevinmatthes** [comment]
The automation of version number incrementing is a common task for which plenty of tools are available.  Here are some I know about / have heard of:

Name | Language | Maintenance Status | Version Number Incrementing | Release Date Injection
-|:-:|-:|:-:|:-:
[Aeruginous](https://github.com/kevinmatthes/aeruginous-rs) | Rust | actively maintained | yes | yes
[bump2version](https://github.com/c4urself/bump2version) | Python 3 | ⚠️ unmaintained | yes | no
[cargo-release](https://github.com/crate-ci/cargo-release) | Rust | actively maintained | yes | yes
[tbump](https://github.com/your-tools/tbump) | Python 3 | actively maintained | yes | yes

I worked with bump2version for years but then it became unmaintained, so I had to look for a new tool.  I decided to implement my own solution into Aeruginous with the result that it specialises in CITATION.cff and Rust manifest updates.

cargo-release is the tool I came across when setting up CITATION.cff for Clap but I consider it rather complex such that a rather learning effort might be required.  It is controlled via Cargo.toml.

tbump is one of the Python 3 replacements I found when looked for a possible successor of bump2version.  I never worked with the tool but it appears to be rather similar to bump2version.  Furthermore, it seems to also allow for release date injection as part of its extended features (`before_commit`).
